vcluster/configure/vcluster-yaml/sync/from-host/custom-resources.mdx

```diff
@@ -176,17 +207,13 @@ sync:
         scope: Namespaced
         mappings:
           byName:
-            "default/my-cm": "barfoo2/cm-my"
+            "production/web-cert": "certificates/prod-web-cert"
```

Comment:
my-cm most likely stands for `ConfigMap` so we need to keep it but make better name for the firstpart.